packages/react-components/src/components/Picker/PickerList.spec.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { render, fireEvent, vi } from 'test-utils';
+import noop from '../../utils/noop';
+import { IPickerListProps, PickerList } from './PickerList';
+import styles from './PickerList.module.scss';
+
+const baseClass = 'picker-list';
+const itemClassName = `${baseClass}__item`;
+
+const defaultProps = {
+  isOpen: false,
+  items: [
+    { key: 'one', name: 'Option one' },
+    { key: 'two', name: 'Option two' },
+    { key: 'three', name: 'Option three' },
+    { key: 'four', name: 'Option four' },
+    { key: 'five', name: 'Option five' },
+    { key: 'six', name: 'Option six' },
+    { key: 'seven', name: 'Option seven' },
+  ],
+  selectedItem: null,
+  onClose: () => noop,
+  onSelect: () => noop,
+};
+
+const renderComponent = (props: IPickerListProps) => {
+  return render(<PickerList {...props} />);
+};
+
+describe('<PickerList> component', () => {
+  it('shouldnt render PickerList if not isOpen', () => {
+    const { container } = renderComponent(defaultProps);
+
+    expect(container.firstChild).toBeNull();
+  });
+
+  it('should render PickerList if isOpen', () => {
+    const { container } = renderComponent({
+      ...defaultProps,
+      isOpen: true,
+    });
+
+    expect(container.firstChild).toBeVisible();
+  });
+
+  it('should call onSelect when list item clicked', () => {
+    const mockedFunction = vi.fn();
+    const { getByText } = renderComponent({
+      ...defaultProps,
+      isOpen: true,
+      onSelect: mockedFunction,
+    });
+
+    fireEvent.click(getByText('Option three'));
+    expect(mockedFunction).toHaveBeenCalledWith({
```

Comment:
It is way better to express the expected meaning of the test. Here the goal is to check id `onSelect` has been called. So I would change the naming a little bit. WDYT?

```suggestion
    const onSelect = vi.fn();
    const { getByText } = renderComponent({
      ...defaultProps,
      isOpen: true,
      onSelect,
    });

    fireEvent.click(getByText('Option three'));
    expect(onSelect).toHaveBeenCalledWith({
```